.gitignore

```diff
@@ -42,3 +42,7 @@ push_csvs.py
 
 # Data dump for docker postgres
 docker_postgres_dump.sql
+
+# Dataset CSV
+app/github_public_repo/serotracker_dataset.csv
```

Comment:
you can replace these two lines with **/serotracker_dataset.csv which will generalize to both cases

Reply:
Oh that's awesome, didn't know about that. Thanks!